src/adaPets.js

```diff
@@ -25,11 +31,17 @@ const removePet = (selectedPetId) => {
     setError("You tried to remove a pet without selecting it!");
   } else {
     // Fill out as part of Wave 3.
-  }
+    axios.delete(BASE_URL + selectedPetId)
+    .then((result) => {setResult(result.data)})
+    .catch((error) => {setError(`Unable to remove pet: ${error}`)})
+  };
 };
 
 const addPet = (petInfo) => {
```

Comment:
Note from reading the test, `addPet` expects petInfo to be formatted as a nested array:
```
petInfo = {
   name: "a name"
   options: {
       optional features
   }
}
```

As such, this data structure needs to be flattened before it is passed to the axios call. One was to do this is with the code block below.
```
petInfo = petInfo.options
petInfo.name = petInfo.name
```